mqttx-web/umbrel-app.yml

```diff
@@ -0,0 +1,22 @@
+manifestVersion: 1
+id: mqttx-web
+name: MQTTX Web
+tagline: Your All-in-One MQTT Client Toolbox
+icon: ""
+category: automation
+version: "1.11.0"
+port: 1884
+description: >-
+  MQTTX Web is an open-source MQTT 5.0 WebSocket client on the browser to help you debug your MQTT services and applications faster.
+
+  The user interface (UI) of MQTTX adopts a chat-based layout, simplifying operational logic. It enables users to establish multiple M>
```

Comment:
It looks like you may have accidentally cut-off the app description. Right now it ends with `...multiple M>`.

Also, to have the whitespace render nicely in the app store, there needs to be 2 blank lines between paragraphs. With just one blank line, it will render like this:

<img width="843" alt="image" src="https://github.com/user-attachments/assets/8d4b48d4-54fd-4321-b170-05bb31e20f9b">

Reply:
fixed: added linefeed and missing text.